src/App.jsx

```diff
@@ -20,11 +20,37 @@ function App() {
     })
   }, [])
 
-  useEffect(() => {
-    if (!isLoading) {
-      localStorage.setItem('savedTodoList', JSON.stringify(todoList));
+  const fetchData = async () => {
+    const options = {
+      method: 'GET',
+      headers: {
+        Authorization: `Bearer ${import.meta.env.VITE_AIRTABLE_API_TOKEN}`
+      }
+    };
+    const url = `https://api.airtable.com/v0/${import.meta.env.VITE_AIRTABLE_BASE_ID}/${import.meta.env.VITE_TABLE_NAME}`;
+    try {
+      const response = await fetch(url, options);
+      if (!response.ok) {
+        throw new Error(`Error: ${response.status}`);
+      }
+      const data = await response.json();
+      const todos = data.records.map((arr) => {
```

Comment:
Does `arr` mean array? 
When you map the records in the array, each of the records is an object,  not an array. 

You can use `todo` instead of `arr`. Or `obj`. 

You can check by using`typeof` operator.